katello.gemspec

```diff
@@ -42,7 +42,7 @@ Gem::Specification.new do |gem|
   # Pulp
   gem.add_dependency "runcible", ">= 2.11.0", "< 3.0.0"
   gem.add_dependency "anemone"
-  gem.add_dependency "pulpcore_client"
+  gem.add_dependency "pulpcore_client", "3.0.0rc2.dev.1560272824"
```

Comment:
Is this an intended change in the PR?

Reply:
Yes, this is due to how the pulpcore client versions were specified by the pulp3 team. The labels are considered to be "earlier" than than a previous version (May 20, 2019 release date) labelled as "3.0.0rc2", despite a June 11 2019 release date. Until the pulp3 removes the mislabelled gem versions, we have to pin the pulpcore client.